include/SDL_events.h

```diff
@@ -1151,6 +1151,40 @@ extern DECLSPEC Uint8 SDLCALL SDL_EventState(Uint32 type, int state);
  */
 extern DECLSPEC Uint32 SDLCALL SDL_RegisterEvents(int numevents);
 
+typedef void (*SDL_ModalLoopCallback)(void*);
+
+/**
+ * Set a callback to be called during a window modal loop.
+ *
+ * This can be used to run some logic and keep the app running when it is stuck
+ * in a modal loop.
+ *
+ * \param callback the function to call in the window modal loop
+ * \param userdata a pointer that is passed to `callback`
+ *
+ * \since This function is available since SDL 2.0.18.
+ *
+ * \sa SDL_SetModalLoopResizeCallback
+ */
+extern DECLSPEC void SDLCALL SDL_SetModalLoopCallback(SDL_ModalLoopCallback callback, void* userdata);
+
+typedef void (*SDL_ModalLoopResizeCallback)(SDL_Window*, int, int, void*);
```

Comment:
This should be `(SDLCALL * SDL_ModalLoopResizeCallback)` like other public SDL func pointers